openmetadata-docs/content/v1.2.x/connectors/ingestion/workflows/data-quality/tests.md

```diff
@@ -270,19 +270,22 @@ parameterValues:
 ```
 
 ### Table Custom SQL Test
-Write you own SQL test. The test will pass if the following condition is met:
-- The query result return 0 row
+Write you own SQL test. When writting your query you can use 2 strategies:
+- `ROWS` (default): expects the query to be written as `SELECT <field>, <field> FROM <foo> WHERE <condition>`. **Note** if your query returns a large amount of rows it might cause an "Out Of Memeory" error. In this case we recoomend you to use the `COUNT` startegy.
+- `COUNT`: expects the query to be written as `SELECT COUNT(<field>) FROM <foo> WHERE <condition>`.
```

Comment:
add a section for threshold and on how to think abou writing a test.
There is docs below, but looks everyone overlooks those. So to be make it clear,lets add it here